db/filedb.go

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
multiline comments are not that common in go, generally you use one liner `//` and if you need to extend, another line with `//` would be enough